files/migrations.py

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/python3
+"""
+Runs migration scripts for this project
+
+Intended to be triggered by the %post rpm scriptlet rather than manually.
```

Comment:
The script sometimes exits status code 2 and once with 3. Is that intended? ~~I remember you mentioning that it actually has to exit status code 0 (which is what https://docs.fedoraproject.org/en-US/packaging-guidelines/Scriptlets/#_syntax linked below says too).~~ OK I made it to the spec file which has `||:`, got it.

If it's intentional to use 2 and 3 as exit codes, would be nice to document why each one is supposed to be used.

Reply:
I added a docstring to `main()`, hope that helps!